src/procmon.cpp

```diff
@@ -25,7 +25,7 @@ int main(int argc, char *argv[])
     * This is due to a conflict that leads to the consumer thread dieing when the system
     * function is executed.
     */
-    system("setterm -cursor off");
+    std::cout << "\33[?25l";
```

Comment:
Since Procmon for Linux already has a dependency on ncurses, why not use curs_set(0) to hide and curs_set(1) to show?

Reply:
Right, fair point. When I made the PR, I was also hesitating, about whether should I use the ncurses version of the function or just print it directly. I'll update the PR.